src/systems/thermal/Thermal.cc

```diff
@@ -48,14 +51,59 @@ void Thermal::Configure(const Entity &_entity,
     gazebo::EntityComponentManager &_ecm,
     gazebo::EventManager & /*_eventMgr*/)
 {
-  if (!_sdf->HasElement("temperature"))
+  const std::string temperatureTag = "temperature";
+  const std::string heatSignatureTag = "heat_signature";
+  const std::string minTempTag = "min_temp";
+  const std::string maxTempTag = "max_temp";
+
+  if (_sdf->HasElement(temperatureTag) && _sdf->HasElement(heatSignatureTag))
+  {
+    ignerr << "Failed to load thermal system. "
```

Comment:
You could change this to warning, and restructure the if to prioritize the `heatSignatureTag`. This would allow the plugin to work, even if both are specified.

Reply:
I made the update to prefer the heat signature over temperature if they are both specified in https://github.com/ignitionrobotics/ign-gazebo/pull/498/commits/a60e5d37ecc39090ce8141472f82b693c8466751